HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -119,14 +119,24 @@ def customiseFor10418(process):
        setattr(process.hltBSoftMuonMu5L3,'algorithmMaskContains', cms.vstring())
     return process
 
+# migrate RPCPointProducer to a global::EDProducer (PR #10927)
+def customiseFor10927(process):
+    if any(module.type_() is 'RPCPointProducer' for module in process.producers.itervalues()):
+        if not hasattr(process, 'CSCObjectMapESProducer'):
+            process.CSCObjectMapESProducer = cms.ESProducer( 'CSCObjectMapESProducer' )
+        if not hasattr(process, 'DTObjectMapESProducer'):
+            process.DTObjectMapESProducer = cms.ESProducer( 'DTObjectMapESProducer' )
+    return process
+
 # CMSSW version specific customizations
-def customiseHLTforCMSSW(process,menuType="GRun",fastSim=False):
+def customiseHLTforCMSSW(process, menuType="GRun", fastSim=False):
     import os
     cmsswVersion = os.environ['CMSSW_VERSION']
 
     if cmsswVersion >= "CMSSW_7_6":
         process = customiseFor10418(process)
```

Comment:
how do the lines from customiseFor10927 make it to the 76X setup?
Are they just not needed in the tests running there ? (jenkins tests in 76X appear to pass)